hazelcast/src/main/java/com/hazelcast/query/impl/BitmapIndexStore.java

```diff
@@ -125,6 +125,8 @@ public void insert(Object value, QueryableEntry entry, IndexOperationStats opera
             return;
         }
 
+        Iterator statValues = makeIterator(value);
```

Comment:
Probably passing the `IndexOperationStats` instance to `Bitmap` methods might be a better alternative. Yes, it kind of breaks the separation of responsibilities between `BitmapIndexStore` and `Bitmap`, but we would not need extra iterators allocation and we would iterate the entries only once.